react/features/e2ee/components/E2EELabel.js

```diff
@@ -29,16 +29,13 @@ class E2EELabel extends Component<Props> {
             return null;
         }
 
-        // eslint-disable-next-line react/jsx-max-props-per-line
-        const icon = <Icon size = { 22 } src = { IconE2EE } />;
```

Comment:
how about the size now?

Reply:
It now uses the default size, and I think that looks great.